src/fiatconnect/saga.ts

```diff
@@ -893,97 +893,128 @@ export function* handleFetchFiatConnectProviders() {
   }
 }
 
-export function* handleCreateFiatConnectTransfer({
+/**
+ * Initiates a fiatconnect transfer using the fiatconnect SDK to call a transfer/out or transfer/in endpoint
+ * for a provider.
+ **/
+export function* _initiateFiatConnectTransferWithProvider({
```

Comment:
nit: Would `_initiateFiatConnectTransferWithProvider` suffice? I saw Joe's earlier comment, but I think "FiatConnect" is redundant here since this is in the fiatconnect/saga file and the `withProvider` adds context that this is something to do with FC provider.